jaeger_client/tracer.py

```diff
@@ -148,6 +150,11 @@ def start_span(self,
         if self.active_span is not None and not ignore_active_span:
             parent = self.active_span
 
+        tags = tags or {}
+        is_local_root = not parent or isinstance(parent, SpanContext)
```

Comment:
parent being a context isn't necessarily indicative of this being a local root, but I'm not seeing a way of supporting that api usage that doesn't involve a store of trace ids whose value would be minimal and would likely require some ttl support.

Reply:
Yes, I do realize that but like you said there isn't a great way to detect this at the moment and this is sort of the best effort we can make. We could modify the rest of the tracer to add `is_remote` flag to the SpanContext class and set it when context is extracted by one of the propagation implementations. I think that should be more reliable but still not sure if it'd be worth the effort.